### DISCUSSION TITLE
milvus 2.5全文检索，分词器情况

### DISCUSSION THREAD

**Discussion by kxleee** [milvus-io/milvus#38199] [Ideas & Feature requests]
看到milvus的2.5版本集成了分词器，请问能分享分词器的详细介绍信息吗？分词模式，分词速度，分词效果等，是否可插入自定义分词器，感谢

**Comment by yhmo** [comment]
看一下文档这章的内容是否有帮助：https://milvus.io/docs/analyzer-overview.md
